modules/cnf-programming-the-sriov-fec-operator-acc100.adoc

```diff
@@ -0,0 +1,345 @@
+   - nodeName: node1 <2>
+     physicalFunctions:
+       - pciAddress: 0000:af:00.0 <3>
+         pfDriver: "pci-pf-stub"
+         vfDriver: "vfio-pci"
+         vfAmount: 16 <4>
+         bbDevConfig:
+           acc100:
+             # Programming mode: 0 = VF Programming, 1 = PF Programming
+             pfMode: false
+             numVfBundles: 16
+             maxQueueSize: 1024
+             uplink4G:
+               numQueueGroups: 0
+               numAqsPerGroups: 16
+               aqDepthLog2: 4
+             downlink4G:
+              numQueueGroups: 0
+              numAqsPerGroups: 16
+              aqDepthLog2: 4
+             uplink5G:
+              numQueueGroups: 4
+              numAqsPerGroups: 16
+              aqDepthLog2: 4
+             downlink5G:
+              numQueueGroups: 4
+              numAqsPerGroups: 16
+              aqDepthLog2: 4
+----
+<1> Specify a name for the CR object. The only name that can be specified is `config`.
+<2> Specify the node name.
+<3> Specify the PCI address of the card on which the SR-IOV-FEC Operator will be installed.
+<3> Specify the number of virtual functions to create. For the Intel® vRAN Dedicated Accelerator ACC100, create all 16 VFs.
++
+[NOTE]
+====
+The card is configured to provide up to 8 queue groups with up to 16 queues per group. The queue groups can be divided between groups allocated to 5G and 4G and Uplink and Downlink.
+The Intel® vRAN Dedicated Accelerator ACC100 can be configured for:
+
+* 4G or 5G only
+* 4G and 5G at the same time
+
+Each configured VF has access to all the queues. Each of the queue groups have a distinct priority level. The request for a given queue group is made from the application level that is, the vRAN application leveraging the FEC device.
+====
+
+.. Apply the CR.
++
+[source,terminal]
+----
+$ oc apply -f sriovfec_acc100cr.yaml
+----
++
+After creation of the CR, the SR-IOV FEC daemon starts configuring the FEC device.
+
+.. Check the status:
++
+[source,terminal]
+----
+$ oc get sriovfecclusterconfig config -o yaml
+----
++
+.Example output
+[source,yaml]
+----
+status:
+    conditions:
+    - lastTransitionTime: "2021-03-19T11:46:22Z"
+      message: Configured successfully
+      observedGeneration: 1
+      reason: Succeeded
+      status: "True"
+      type: Configured
+    inventory:
+      sriovAccelerators:
+      - deviceID: 0d5c
+        driver: pci-pf-stub
+        maxVirtualFunctions: 16
+        pciAddress: 0000:af:00.0
+        vendorID: "8086"
+        virtualFunctions:
+        - deviceID: 0d5d
+          driver: vfio-pci
+          pciAddress: 0000:b0:00.0
+        - deviceID: 0d5d
+          driver: vfio-pci
+          pciAddress: 0000:b0:00.1
+        - deviceID: 0d5d
+          driver: vfio-pci
+          pciAddress: 0000:b0:00.2
+        - deviceID: 0d5d
+          driver: vfio-pci
+          pciAddress: 0000:b0:00.3
+        - deviceID: 0d5d
+          driver: vfio-pci
+          pciAddress: 0000:b0:00.4
+----
```

Comment:
Please repeat the callouts like you did for the previous example.  Customers should check the `message` field for "Configured successfully" and so on, right?